backend/FwLite/MiniLcm/Models/PartOfSpeech.cs

```diff
@@ -3,7 +3,8 @@
 public class PartOfSpeech : IObjectWithId
 {
     public Guid Id { get; set; }
-    public MultiString Name { get; set; } = new();
+    public virtual MultiString Name { get; set; } = new();
+    // TODO: Probably need Abbreviation in order to match LCM data model
```

Comment:
I feel like leaving Abbreviation for a later PR; let me know if you want it in this one.

Reply:
later is fine